cognite/extractorutils/rest/extractor.py

```diff
@@ -81,7 +82,7 @@ def __init__(
             use_default_state_store=use_default_state_store,
             config_class=config_class,
         )
-        self.base_url = base_url or ""
+        self._default_base_url = base_url or ""
```

Comment:
is default empty string?

Reply:
Yes, if no base url is given the default is empty string (so the `urljoin` later still works, but does nothing). A user would then have to specify a full URL in the decorator, like so:

``` python
@extractor.get("https://my.source/api/whatever", ...)
def get_stuff(...)
```